tests/securedrop/install_workstation.pm

```diff
@@ -16,6 +16,55 @@ use strict;
+    assert_script_run('qvm-run -p work -- sudo rpmkeys --import securedrop-release-key.pub');
+    assert_script_run('qvm-run -p work -- "echo -e \"[sd]\nenabled=1\nbaseurl=https://yum-qa.securedrop.org/workstation/dom0/f37\nname=boostrap\"  | sudo tee /etc/yum.repos.d/securedrop-temp.repo"');
+    assert_script_run('qvm-run -p work -- dnf download -y securedrop-workstation-dom0-config');
+    assert_script_run('qvm-run -p work -- "rpm -Kv securedrop-workstation-dom0-config-*.rpm"');  # TODO confirm output is correct
+    assert_script_run('qvm-run -p work -- "cat /home/user/securedrop-workstation-dom0-config-*.rpm" > securedrop-workstation.rpm');
+    assert_script_run('sudo dnf -y install securedrop-workstation.rpm');
+};
+
+sub install_dev {
+    # Assumes terminal window is open
+
+
+    assert_script_run('qvm-check sd-dev || qvm-create --label gray sd-dev --class StandaloneVM --template debian-12-xfce');
+
+    # Building SecureDrop Workstation RPM and installing it in dom0
+    assert_script_run('qvm-run -p sd-dev "git clone https://github.com/freedomofpress/securedrop-workstation"');
+    assert_script_run('qvm-run -p sd-dev "git -C securedrop-workstation checkout ' . get_var('GIT_REF') . '"');
+
+    # SecureDrop dev. env. according to https://developers.securedrop.org/en/latest/setup_development.html
+    # DOCKER INSTALL according to https://docs.docker.com/engine/install/debian/
+    assert_script_run('qvm-run -p sd-dev "sudo apt-get update"');
+    assert_script_run('qvm-run -p sd-dev "sudo apt-get install -y ca-certificates curl"');
+    assert_script_run('qvm-run -p sd-dev "sudo install -m 0755 -d /etc/apt/keyrings"');
+    assert_script_run('qvm-run -p sd-dev "sudo curl -fsSL https://download.docker.com/linux/debian/gpg -o /etc/apt/keyrings/docker.asc"');
+    assert_script_run('qvm-run -p sd-dev "sudo chmod a+r /etc/apt/keyrings/docker.asc"');
+    assert_script_run('qvm-run -p sd-dev ". /etc/os-release && echo \"deb [arch=amd64 signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/debian \$VERSION_CODENAME stable\" | sudo tee /etc/apt/sources.list.d/docker.list \> /dev/null"');
+    assert_script_run('qvm-run -p sd-dev "sudo apt-get update"');
+    assert_script_run('qvm-run -p sd-dev "sudo apt-get install -y docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin"');
+    assert_script_run('qvm-run -p sd-dev "sudo groupadd docker || true"');
+    assert_script_run('qvm-run -p sd-dev "sudo usermod -aG docker \$USER"');
+    assert_script_run('qvm-kill sd-dev && qvm-start sd-dev');  # Restart for groupadd to take effect
+
+    # Also copy to dom0 to run tests later, but no need to configure env vars for future `make clone`.
+    assert_script_run("qvm-run --pass-io sd-dev 'tar -c -C /home/user/ securedrop-workstation' | tar xvf -", timeout=>300);
+    assert_script_run("ls");
+
+    assert_script_run('qvm-run -p sd-dev "sudo apt install -y make git jq"');
```

Comment:
```suggestion
    assert_script_run('qvm-run -p sd-dev "sudo apt-get install -y make git jq"');
```

In general always use `apt-get` in scripting (`apt` is for humans).

Also any reason this can't be merged into the earlier installation of ca-certificates/curl?